core/util/src/main/java/org/eclipse/rdf4j/common/io/FileUtil.java

```diff
@@ -9,307 +9,12 @@
 package org.eclipse.rdf4j.common.io;
 
 import java.io.File;
-import java.io.FileInputStream;
 import java.io.IOException;
-import java.nio.file.Files;
-import java.util.Collection;
-import java.util.StringTokenizer;
 
 /**
  * Utility methods for operations on Files.
  */
 public class FileUtil {
-
-	private static final String ILLEGAL_FILE_NAME_CHARS = "\\/:*?\"<>|";
-
-	/**
-	 * Gets the relative representations of a file compared to another.
-	 *
-	 * @param subj       The File to find the relative form for.
-	 * @param relativeTo The File 'subj' should be made relative to.
-	 * @return The relative representation of subj.
-	 */
-	public static String getRelativePath(File subj, File relativeTo) {
```

Comment:
There are several public methods here that were never marked deprecated. I know we're aiming for a major so backward compatibility is less of an issue, but perhaps we should keep these in place marked deprecated.

Reply:
As mentioned below, removing them (and add them back into the code if we receive complaints)